scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -43,17 +43,18 @@ RUN wget https://github.com/Kitware/CMake/releases/download/v3.14.5/cmake-3.14.5
 FROM base AS libraries
 
 # Boost
-RUN git clone -b boost-1.69.0 https://github.com/boostorg/boost.git \
-    /usr/src/boost; \
-    cd /usr/src/boost; \
-    git submodule update --init --recursive; \
+RUN wget -q 'https://dl.bintray.com/boostorg/release/1.73.0/source/boost_1_73_0.tar.bz2' -O boost.tar.bz2; \
```

Comment:
Does the tarball equivalent to a submodule clone? I'm worried that we may not build all the boost libraries we need.

Reply:
We hardly need any of them :-). We only build those specifically enabled in the ``b2`` command below, i.e. ``system filesystem unit_test_framework program_options``... and if those weren't in the tarball, the build would fail.

Reply:
Or is there reason to believe that the fuzzer builds will require any more boost libraries than the normal solidity builds?

Reply:
Or put differently: since the tarball and these libraries are enough for ``Dockerfile.emscripten``, it should be enough here, shouldn't it?

Reply:
I see. I wasn't aware of the lines in the emscripten Dockerfile. In that case, this should work as expected.